pages/vi/vi-docker-tutorial.md

```diff
@@ -254,4 +254,13 @@ Commands:
   version            Show the Docker-Compose version information
 ```
 
+## Useful Links
+
+[Why install Docker?](https://www.docker.com/why-docker) 
```

Comment:
`https://www.docker.com/what-docker` gives idea of what docker is

Reply:
will do